stubs/PyMySQL/pymysql/connections.pyi

```diff
@@ -195,6 +196,8 @@ class Connection(Generic[_C]):
     def get_proto_info(self): ...
     def get_server_info(self): ...
     def show_warnings(self): ...
+    def __enter__(self: Self) -> Self: ...
+    def __exit__(self, *exc_info: Any) -> None: ...
```

Comment:
```suggestion
    def __exit__(self, *exc_info: object) -> None: ...
```
Prefer object over Any when the function accepts all objects.

Reply:
Slowpoke 😁

Reply:
I failed today